charts/rancher-eks-operator/1.0.101/Chart.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+name: rancher-eks-operator
+description: A Helm chart for provisioning EKS clusters
+home: https://github.com/rancher/eks-operator
+sources:
+  - "https://github.com/rancher/eks-operator"
+version: 1.0.101
+appVersion: v1.0.1
```

Comment:
We might want to remove eks-operator from 2.4 line eventually since we made a 2.5 line, but its fine for now since questions.yaml is in place.